director/cache_ads.go

```diff
@@ -442,15 +442,25 @@ func updateLatLong(ad *server_structs.ServerAd) error {
 	return nil
 }
 
-// Apply downtimes provided by Origin/Cache servers to the director's in-memory state for the given server.
+// Apply downtimes provided by Origin/Cache servers (<= v7.21) to the director's in-memory state for the given server.
 // This function treats a nil or empty slice as "no downtimes", clearing any stale
 // serverFiltered entry and removing cached server downtimes.
 func applyServerDowntimes(serverName string, downtimes []server_structs.Downtime) {
+	// If the server is >= v7.22, it sets the serverID in the downtime entry.
```

Comment:
Can you add a loud TODO to indicate this can get removed when all listeners are above this version?